### PR TITLE
Piglin Barter Fix

### PR DESCRIPTION
**Pull request by TheAbsolutionism**
### Description

This PR aims to fix the piglin barter test failing JUnit locally on Windows.
(feature branch as the patch branch does not contain it)
---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Review by APickledWalrus — APPROVED** [review]

**Comment by sovdeeth** [comment]
Pretty sure renaming that file breaks tests, you should undo that

**Comment by TheAbsolutionism** [comment]
Interesting, I didnt rename the file

**Comment by sovdeeth** [comment]
Oh wait this is feature 
You can just remove all that nonsense then, there's no worries about versioning anymore

**Comment by TheAbsolutionism** [comment]
@sovdeeth Wait, what nonsense are you talking about? lol

**Review by Moderocky — APPROVED** [review]

**Review by Efnilite — APPROVED** [review]

File: src/test/skript/junit/EvtPiglinBarter.sk [hunk 1]
```diff
@@ -58,4 +71,4 @@ on piglin barter:
 	if size of barter drops is 0:
 		complete objective "remove multiple items from barter drops" for {_test}
 
-	complete objective "barter drops test complete" for {_test}
+	complete objective "barter drops test complete" for {_test}
```
Comment (sovdeeth) [inline src/test/skript/junit/EvtPiglinBarter.sk:74]:
```suggestion
	complete objective "barter drops test complete" for {_test}

```

**Pull request merged**